keycloak-extensions/src/test/java/org/alvearie/utils/SeleniumOauthInteraction.java

```diff
@@ -456,22 +305,24 @@ public SeleniumOauthInteraction() {
     }
 
     // main method allows for easy standalone testing. This could be moved to a separate file.
-    public static void main(String[] args) throws UnsupportedEncodingException, URISyntaxException {
-        SeleniumOauthInteraction s = new SeleniumOauthInteraction();
-        s.appidUser = "a";
-        s.appidPwd = "a";
-        s.driverPath = "/usr/local/bin/geckodriver";
-        s.browserPath = "/Applications/Firefox.app/Contents/MacOS/firefox";
-
+    public static void main(String[] args) throws Exception {
         String realm = "test";
-        String client = "test";
+        String baseUrl = "https://localhost:8080/auth/realms/" + realm + "/protocol/openid-connect/";
+
+        SeleniumOauthInteraction s = new SeleniumOauthInteraction();
+        s.keycloakUser = "a";
+        s.keycloakPwd = "a";
+        s.appClientId = "test";
+        s.appRedirectUri = "https://localhost";
+        s.oauthAuthUrl = baseUrl + "/auth";
```

Comment:
Extra slash when concatenated to baseUrl.

Reply:
good catch
```suggestion
        s.oauthAuthUrl = baseUrl + "auth";
```

Reply:
fixed